conf/datasources.ini.sample.finna

```diff
@@ -0,0 +1,4 @@
+; Finna specific datasource settings
+;[marc]
+;Which fields to include as a linking id for record as comma separated list. Default 001,035,999.
+;driverParams[] = "recordLinkingIdFields=001,035,999"
```

Comment:
Tämän voisi tehdä niin, että tässä listassa määritellään myös osakenttä. Eli tyyliin:
`;driverParams[] = "recordLinkingIdFields=001,035a,999c"`
Näin näkee suoraan mistä osakentästä tieto tulee, eikä Marc-luokan tarvitse tietää kaikkia vaihtoehtoja etukäteen.

Näiden lisäksi tarvitaan erikoistapaus 003+001, koska monissa tapauksessa linking_id on muotoa `(003)001`. Eli vastaava kuin `003InLinkingID`-asetus on ollut. Ks. esim. [osakohde](https://kansalliskirjasto.finna.fi/Record/fikka.4378875), jossa 773w:ssä on `(FI-NLD)888209` eli [emokohteen](https://kansalliskirjasto.finna.fi/Record/fikka.5028532) linking_id:n pitäisi olla se eikä pelkkä `888209`.